src/server/arenaApp.ts

```diff
@@ -1,9 +1,19 @@
 import { Express, RequestHandler } from 'express'
-
+import { User as _User } from '@openforis/arena-core'
 /**
  * Express app wrapper.
  */
 export interface ArenaApp {
   express: Express
   session: RequestHandler
 }
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
this is not good. better to add 
```
rules: {
  "@typescript-eslint/no-namespace": "off"
}
```
?

Reply:
Are we allowed to use it elsewhere also in this case?

Reply:
you can use export instead of namespace.